cypress/e2e/2-createColony.js

```diff
@@ -1,21 +1,25 @@
 describe('Create a new colony', () => {
   if (!Cypress.config().skipInitTests) {
     it('creates a new colony with new token', () => {
-      const { name, nativeToken } = Cypress.config().colony;
+      const { colony, baseUrl } = Cypress.config();
```

Comment:
Does the config need to be called every time, or can we do this somewhere at the start of the file, somewhere inside the `describe` declaration, but outside of any individual `it` declarations

Not major, just a thought

Reply:
Yeah, I already thought about it. But it seems that Arren has already refactored it everywhere. I need to wait until he merges his PR first so that I can make the final pass at all the tests. And then I'll make sure that it's consistent across all the files.